integration-tester/tester/logic.go

```diff
@@ -0,0 +1,157 @@
+package tester
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/abetterinternet/prio-server/integration-tester/kubernetes"
+	m "github.com/abetterinternet/prio-server/manifest-updater/manifest"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (t *Tester) Start() error {
+	manifest, err := GetManifest(t.manifestFileUrl)
+	if err != nil {
+		return err
+	}
+	bsk, err := t.getValidBSK(manifest)
+	if err != nil {
+		return err
+	}
+
+	pdk, err := t.getValidPDK(manifest)
+	if err != nil {
+		return err
+	}
+
+	job := t.createJob(manifest, bsk, pdk)
+
+	return kubernetes.ScheduleJob(t.namespace, job)
+}
+
+func (t *Tester) createJob(manifest *m.DataShareSpecificManifest, bsk, pdk *corev1.Secret) *batchv1.Job {
```

Comment:
You'll need to introduce something that deletes these jobs when they're completed, or we'll run into the same problems we did with intake jobs where we have too many completed jobs lying around.

Reply:
Yep!